content/espanol/sql_workshop/activity10/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Actividad 10: Aprende el comando SUM"
+description: "Aprende el comando SUM"
+date: 2020-07-07
+prereq: "Comando SELECT, comando WHERE"
+difficulty: "Principiante"
+draft: false
+---
+<!-- Links for javascript and CSS needed for drop down logic -->
+<link rel="stylesheet" href="../default/_default.css" type="text/css"></link>
+<link rel="stylesheet" href="../default/_type.css" type="text/css"></link>
+<script type="text/javascript" src="../default/_default.js"></script>
+<script type="text/javascript" src="../default/_type.js"></script>
+<script type="text/javascript" src="../default/alasql.js"></script>
+<script type="text/javascript" src="../default/db.js"></script>
+<link rel="stylesheet" href="_activity10.css" type="text/css"></link>
+<script type="text/javascript" src="_activity10.js"></script>
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/UCxPAE0xw3I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```

Comment:
For all the activity pages feel free to delete the youtube embed as it will not apply to the spanish workshop for now

Reply:
Fixed